lib/dry/schema/message_compiler.rb

```diff
@@ -199,6 +199,14 @@ def message_tokens(args)
             hash[arg[0]] = arg[1]
           end
         end
+        args.any? { |e| e.first == :size } ? append_mapped_size_tokens(tokens) : tokens
+      end
+
+      # @api private
+      def append_mapped_size_tokens(tokens)
+        # this is a temporary fix for the inconsistency in the "size" errors arguments
+        mapped_hash = tokens.inject({}) { |h, (k, v)| h[k.to_s.gsub("size", "num").to_sym] = v; h }
```

Comment:
I _think_ this would be a good place to use `transform_keys` if I'm not crossing any wires on the logic here.

```suggestion
        tokens.transform_keys do |key|
          k.to_s.gsub("size", "num").to_sym
        end
```

Reply:
Done, rubocop also removed a comma in one hash, but I guess it's ok

Reply:
That was even a method which was there in a first place, but unfortunately it was introduced in Ruby 2.5 and CLIs for Ruby 2.4 were failing with NoMethodErrors :(

I changed it to the `each_with_object`, I guess it's a little bit less awkward